source/github-helpers/index.ts

```diff
@@ -24,12 +24,13 @@ Example tag content on private repositories https://github.com/private/private/c
 export const getCurrentBranch = (): string | undefined => {
 	// .last needed for #2799
 	const feedLink = select.last('link[type="application/atom+xml"]');
+	const findLink = pageDetect.isPRFiles() ? false : select.last('a[data-hotkey="t"]');
 	// The feedLink is not available on `isIssue` #3641
-	if (!feedLink) {
+	if (!(feedLink && findLink)) {
 		return;
 	}
 
-	return new URL(feedLink.href)
+	return new URL(feedLink.href ?? decodeURIComponent(findLink.href))
```

Comment:
??

Both are elements, why would .href be encoded only in one

Reply:
`https://github.com/yakov116/TestR/tree/this/branch/has/many/slashes` => https://github.com/yakov116/TestR/find/this%2Fbranch%2Fhas%2Fmany%2Fslashes

Reply:
YO, YOU SERIOUS?

This means that the branch name with slashes is FINALLY encoded so we don’t need this whole thing at all?!

Reply:
Apparently they still don't do it in most cases, but I'm hopeful.

Related: https://github.com/fregante/shorten-repo-url/issues/27